src/main/resources/network/brightspots/rcv/config_file_documentation.txt

```diff
@@ -113,10 +113,59 @@ Config file must be valid JSON format. Examples can be found in the test_data fo
     set of configuration parameters that specify the tabulation rules
     the "rules" section contains the following parameters:
 
+      "randomSeed" required when tiebreakMode is random, previousRoundCountsThenRandom, or generatePermutation
+        the integer seed for the application's pseudorandom number generator
+        value: [0..2147483647]
```

Comment:
It would be more robust and safer to allow larger seeds, ala the 48 bits actually used by Java, or the 20 decimal digits typical for election auditing approaches.

Reply:
Sure, we can do this.